CHANGELOG.md

```diff
@@ -5,78 +5,174 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 The crates in this repository do not adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) at this time.
 
-## Unreleased
-
-### Added
-- Support env overrides for ~all command-line flags.
-  - Flags that take multiple values can be repeated on the command line,
-    or passed as comma-separated values via environment or command-line args.
-
-### Changed
-
-### Rust Dependencies
-* Updated `rust-toolchain` version to newer nightly
-  * enables use of [Generic Associated Types](https://github.com/rust-lang/rust/issues/44265) and [static async fn in traits](https://github.com/rust-lang/rust/issues/91611)
-* Replaced `datatest` with a custom `test_with_data` macro.
-* Replace `structopt` with `clap`.
-* Updated grpcio from 0.9 to 0.10.
-
 ## [1.2.0]
 
 ### Added
 
- - Encrypted Memos ([MCIP #3](https://github.com/mobilecoinfoundation/mcips/pull/3))
- - Recoverable Transaction History ([MCIP #4](https://github.com/mobilecoinfoundation/mcips/pull/4))
- - Consensus/Fog repository merge
+- Fog is now part of this repository
```

Comment:
```suggestion
- Fog is now part of this repository
- Fog-ledger key image checks are now oblivious, mediated by oblivious ram similar to fog view
```

Reply:
maybe we should just pick this one:

https://github.com/mobilecoinfoundation/mobilecoin/pull/1818/files

Reply:
I copy-pasta'd it.